src/frameworks/remix.json

```
@@ -0,0 +1,22 @@
+{
+  "id": "remix",
+  "name": "Remix",
+  "category": "static_site_generator",
+  "detect": {
+    "npmDependencies": ["remix", "@remix-run/netlify"],
+    "excludedNpmDependencies": [],
+    "configFiles": ["remix.config.mjs"]
+  },
+  "dev": {
+    "command": "remix watch # needs to also start the static server",
+    "port": 8888,
```

```suggestion
    "port": 8888,
```

This is my guess, and the CLI will start the static server when the port is missing (after running the `dev` command).

---

Yes, `port` specifies the port opened by the framework dev command. The change in the CLI enabled the static server instead of that port was missing. What it doesn't have is a way to specify the port of the netlify dev server. Remix sets it to 3000 to match their other docs, but we don't have a way to specify this in framework-info